src/modules/public/i18n/reducer.ts

```diff
@@ -6,7 +7,7 @@ export interface LanguageState {
 }
 
 const defaultLanguage = {
-    code: 'en',
+    code: languages[0],
```

Comment:
```suggestion
    code: languages[0] || 'en',
```

Reply:
export const languages = Cryptobase.config.languages && Cryptobase.config.languages.length > 0 ? Cryptobase.config.languages : ['en'];

it is already elaborated in config.ts